s3/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:latest
+FROM journeymidnight/yig:latest
```

Comment:
just to clarify, who will maintain this image (journeymidnight/yig) on dockerhub going ahead ?

Reply:
Thanks rhsakarpos, we will discuss about the docker image, and will clarify about this.